controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -205,6 +206,10 @@ void master_thread::handle_cmdu_1905_1_message(Socket *sd, ieee1905_1::CmduMessa
         handle_cmdu_1905_autoconfiguration_search(sd, cmdu_rx);
         break;
     }
+    case ieee1905_1::eMessageType::AP_AUTOCONFIGURATION_WIFI_SIMPLE_CONFIGURATION_MESSAGE: {
```

Comment:
Maybe we should keep the names as in the standard:
`AP_AUTOCONFIGURATION_WSC_MESSAGE`

Reply:
Maybe, its even a shorter name so will get better look with clang-format, but not related to this PR (this was already merged and reviewed by @arnout )